pentaho-karaf-assembly/pom.xml

```diff
@@ -177,6 +177,7 @@
                 <descriptor>mvn:org.pentaho.adaptive/adaptive-execution-features/${project.version}/xml/features</descriptor>
                 <descriptor>mvn:org.pentaho.di.plugins/file-open-save-plugin/${project.version}/xml/features</descriptor>
                 <descriptor>mvn:org.pentaho.webpackage/pentaho-webpackage/${project.version}/xml/features</descriptor>
+                <descriptor>mvn:pentaho/pentaho-dataservice-server/${project.version}/xml/features</descriptor>
```

Comment:
@joaormf I'm confused... Shouldn't we be using the pentaho-dataservice-client feature in PDI?

Reply:
That's true if by client you only talk about PRD, PME or PMR, which by the way include only the client part.
The previous code added the `<feature>pdi-dataservice</feature>`, which has the server-plugin, and it makes since the PDI also needs the server plugin running inside karaf to be able to be used, for example, in DET.

We now add the feature named `<feature>pentaho-dataservice-server</feature>` which includes the server and client plugins.